packages/components/PersonaCoin/src/PersonaCoin.types.ts

```diff
@@ -1,9 +1,11 @@
 import type { IViewProps } from '@fluentui-react-native/adapters';
-import { ImageProps, ViewProps, ImageURISource, TextProps } from 'react-native';
+import { ImageProps, ViewProps, ImageURISource, TextProps, ColorValue } from 'react-native';
 import { IBackgroundColorTokens, IForegroundColorTokens } from '@fluentui-react-native/tokens';
 import { IRenderData } from '@uifabricshared/foundation-composable';
+import { Palette } from '@fluentui-react-native/theme-types';
 
 export const personaCoinName = 'RNFPersonaCoin';
+export type Color = keyof Palette | ColorValue;
```

Comment:
I think you misunderstood what I meant. Can we not use this type at all, and on line70 and 71, just have those be of type `ColorValue` ?

Reply:
Oops.. on it! Thanks for clarifying that.